openpower/package/openpower-pnor/Config.in

```diff
@@ -12,6 +12,22 @@ config BR2_PACKAGE_OPENPOWER_PNOR
         help
             Utilites for building a targeting binary image
 
+config BR2_OPENPOWER_SECUREBOOT_KEY_TRANSITION
+        string "Type of Transition HW key hash"
+        help
+            Available options [imprint | production]
```

Comment:
Fixed with a choice box for no key transitions,  x-ition to dev keys, and x-ition to prod keys.  Menuconfig looks fantastic!